metadata.json

```
@@ -10,7 +10,7 @@
   "dependencies": [
     {
       "name": "puppetlabs/stdlib",
-      "version_requirement": ">= 4.13.1 < 9.0.0"
+      "version_requirement": ">= 4.13.1 < 10.0.0"
```

```suggestion
      "version_requirement": ">= 9.0.0 < 10.0.0"
```

`stdlib::to_yaml` was namespaced in stdlib 9.0.0.